crates/bevy_input/src/gamepad.rs

```diff
@@ -1750,27 +1885,42 @@ mod tests {
 
     #[test]
     fn test_axis_settings_default_filter_with_old_values() {
+        let threshold = 0.01;
```

Comment:
I chose not to hardcode the values here because the logic is more important than seeing the literal value